FitGoal/FitGoal/Singup&Singin/UserInputValidator.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-class SignUpValidator {
+class UserInputValidator {
```

Comment:
I think you should have two different types here, one for each case. The problem with he current approach is that you have all the signup fields available for sing in, even though those fields aren't needed in this case.

Reply:
Yeah, that was the problem that I'd found as well